server/services/ewelink/lib/models/index.js

```diff
@@ -0,0 +1,19 @@
+const switchBasic = require('./switch_basic');
+const switchTh = require('./switch_th');
+const switchPow = require('./switch_pow');
+const unhandled = require('./unhandled');
+
+module.exports = {
```

Comment:
Is there any to build device dynamically without having to define them all programmatically?

Reply:
I don't think so, because only the uiid of the device defines it. Nothing else (device type, function list, ...) exists.